### ISSUE TITLE
Run rolling squatter at start

### ISSUE THREAD

**Issue by rsto** [cyrusimap/cassandane#65]
A couple of tests in the JMAP test suites require squatter to run on newly created emails. currently these tests run squatter directly. We might want to start a rolling squatter instance, e.g. by adding it to the services.

**Comment by elliefm** [comment]
I don't think we want this across the board, I'm pretty sure at least some of our tests depend on being able to compare state before and after running squatter (and would race against it).  But, maybe as a feature that tests can opt-in or opt-out of with a :magic-function-attribute?
